pallets/claims/src/lib.rs

```diff
@@ -231,7 +233,7 @@ where
 		_len: usize,
 	) -> TransactionValidity {
 		match call.is_sub_type() {
-			Some(Call::claim(signature)) => match Pallet::<T>::validate_claim(who, signature) {
+			Some(Call::claim{ethereum_signature}) => match Pallet::<T>::validate_claim(who, ethereum_signature) {
```

Comment:
why was this renamed? isn't it breaking any API? https://claim.hydradx.io

Reply:
It was not renamed . it is actually called ethereum_signature in the claim extrinsic.

the reason it had to be renamed in the match statement  is that it had to be changed to use  struct pattern syntax. 

so instead of using this:
```rust
Some(Call::claim{ethereum_signature: signature}) => match Pallet::<T>::validate_claim(who, signature)
```

i went with just eithereium signature.